makeflow/src/makeflow.c

```diff
@@ -78,6 +79,7 @@ an example.
 */
 
 #define MAX_REMOTE_JOBS_DEFAULT 100
+#define MF_DONE_FILE "makeflow_done" 
```

Comment:
it is created by makeflow to inform the mesos scheduler that the workflow is done.